### PR TITLE
Feature/dope 214 add backticks to buckets and params

### PR DESCRIPTION
**Pull request by jansigi**

### CODE REVIEW

File: core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/meta/MetaExpression.kt [hunk 1]
```diff
@@ -5,34 +5,47 @@ import ch.ergon.dope.resolvable.expression.TypeExpression
...
         val bucketDopeQuery = bucket.toDopeQuery()
         return DopeQuery(
-            queryString = "META(${
-            when (bucket) {
-                is AliasedBucket -> bucket.alias
-                else -> bucketDopeQuery.queryString
-            }
-            })",
+            queryString = toFunctionQueryString(
+                "META",
```
Comment (pgruntz) [inline core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/meta/MetaExpression.kt:19]: please add named parameters

File: core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/meta/MetaExpression.kt [hunk 2]
```diff
@@ -5,34 +5,47 @@ import ch.ergon.dope.resolvable.expression.TypeExpression
...
-    val id: Field<StringType> = Field("id", toDopeQuery().queryString)
+    val id: Field<StringType> = getMetaField("id")
 
-    val type: Field<StringType> = Field("type", toDopeQuery().queryString)
+    val type: Field<StringType> = getMetaField("type")
 
-    val keyspace: Field<StringType> = Field("keyspace", toDopeQuery().queryString)
+    val keyspace: Field<StringType> = getMetaField("keyspace")
+
+    private fun <T : ValidType> getMetaField(field: String): MetaField<T> = MetaField(field, toDopeQuery().queryString)
```
Comment (pgruntz) [inline core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/meta/MetaExpression.kt:41]: pass the whole dopeQuery and return the parameters as well

Comment (jansigi) [reply core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/meta/MetaExpression.kt]: done

File: core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/meta/MetaExpression.kt (hunk 1 above)
Comment (jansigi) [reply core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/meta/MetaExpression.kt]: done

**Review by pgruntz — APPROVED** [review]

**Pull request merged**